lib/litani.py

```diff
@@ -184,7 +193,7 @@ def cache_pointer_dirs():
 
 def get_cache_dir(path=os.getcwd()):
     try:
-        return _get_cache_dir(path)
+        return CacheDir.get() or _get_cache_dir(path)
```

Comment:
Do you think the following is more clear? 

`return CacheDir(path=path) or _get_cache_dir(path)`

Reply:
We have to return the `path` and not the object right?

Reply:
Yes. Made some changes along with this.